### PR TITLE
Allow multiple tap/hold/doubletap actions.

### PR DESCRIPTION
**Pull request by thomasloven**
## Proposed change

Add a new action type - multiple, which takes a list of `actions:` and perform them all.

Use case is e.g. if you have a layout like this https://www.youtube.com/watch?v=5y6rhwr5Y8c you may want to activate a script and get back to the start view with a single tap.

I was a bit worried about the maintenance burden, but with all the type checking it turns out to be only four lines of business logic in `handle-action.ts` and a single special case to handle in `hui-action-editor.ts` - which is *all* special cases already...

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml
tap_action:
  action: multiple
  actions:
    - action: toggle
    - action: navigate
      navigation_path: /lovelace/default_view
```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by oywino** [comment]
I tried to accomplish exactly this by building a button-card:

`type: button
tap_action:
  action: call-service
    - service: remote.turn_on
      service_data:
        activity: 41122377
      entity_id: remote.harmony
    - service: media_player.play_media
      service_data:
        media_content_id: https://p7.p4groupaudio.com/P07_MH
        media_content_type: music
entity_id: media_player.stue_ac_lms`

But of course it failed. Although to me - the YAML structure I used seem logical, so I actually did expect it to work.

File: src/panels/lovelace/common/handle-action.ts [hunk 1]
```diff
@@ -40,69 +40,74 @@ export const handleAction = async (
...
-  if (
-    actionConfig.confirmation &&
-    (!actionConfig.confirmation.exemptions ||
-      !actionConfig.confirmation.exemptions.some(
-        (e) => e.user === hass!.user!.id
-      ))
-  ) {
-    forwardHaptic("warning");
+  const actionConfigs =
+    actionConfig.action === "multiple" ? actionConfig.actions : [actionConfig];
```
Comment (bramkragten) [inline src/panels/lovelace/common/handle-action.ts:44]: Check if `actions` is an array

Comment (thomasloven) [reply src/panels/lovelace/common/handle-action.ts]: Done

**Comment by thomasloven** [comment]
I'll take this back to the drawing board for a while.

Besides the merge conflict, confirmations can now only be applied to the individual action inside the `multiple` action, not to the `multiple` action itself - which would make more sense if you had to choose one...

File: src/panels/lovelace/common/handle-action.ts [hunk 2]
```diff
@@ -41,7 +41,11 @@ export const handleAction = async (
   }
 
   const actionConfigs =
-    actionConfig.action === "multiple" ? actionConfig.actions : [actionConfig];
+    actionConfig.action === "multiple"
+      ? Array.isArray(actionConfig.actions)
```
Comment (bramkragten) [inline src/panels/lovelace/common/handle-action.ts:45]: <https://github.com/home-assistant/frontend/blob/dev/src/common/ensure-array.ts>

Comment (thomasloven) [reply src/panels/lovelace/common/handle-action.ts:45]: How I googled for this, which I assumed must be in js/ts somewhere. Should have git greped...

**Pull request closed without merging**